### ISSUE TITLE
일일 실습 보고서 제출 (7월 10일, 수)

### ISSUE THREAD

**Issue by csbae** [GoonGuMa/DSC-Oh-Jun-Young#17]
7월10일은 계절학기 13주차에 해당합니다.
프로젝트 개발 내용을 중심으로 일일 실습 보고서를 작성하여 제출해주세요.
특히 오늘 수업 시간에 토의한 내용과 실행 계획을 작성해주세요.
프로젝트 개발 진행에 따라 필요한 경우 자유 양식을 추가 사용하세요.
제출 마감은 7월 10일 (수) 자정입니다.

**Comment by GoonGuMa** [comment]
[7.10 실습.pdf](https://github.com/user-attachments/files/16162337/7.10.pdf)

**Comment by GoonGuMa** [comment]
약 15가지 정도의 하이퍼파라미터를 수정하였고 그 과정을 모두 작성하였습니다.
아직 정확도가 60% 초반대에 머물러있습니다만, 과적합부분은 해결된듯합니다..

**Comment by csbae** [comment]
> 약 15가지 정도의 하이퍼파라미터를 수정하였고 그 과정을 모두 작성하였습니다. 아직 정확도가 60% 초반대에 머물러있습니다만, 과적합부분은 해결된듯합니다..

모델의 구조 및 하이퍼파라미터를 최적화하는 것은 매우 어려운 일입니다.
데이터에 대한 의존성도 높고요.
실험을 하면서 다음 두 가지 사항을 확인해보세요.
- epoch이 증가할 때 train loss가 감소하는지? 
(증가 감소가 반복되더라도 전체적으로 감소하는 경향이면 괜찮습니다만, 그게 아니면 모델이나 하이퍼파라미터가 적절하지 않은 것입니다. 이것은 epoch에 따른 loss 변화를 그림으로 그려보면 쉽게 알 수 있습니다.)
- train loss와 validation loss의 차이가 너무 크지 않은지?
(이 차이가 너무 크면 과적합 상태로 생각할 수 있습니다. 과적합을 해결하는 방법 중 하나는 dropout을 이용하는 것도 있습니다.)

**Comment by GoonGuMa** [comment]
epoch 증가할때 Train loss이 감소하는것도 확인되었습니다,
dropout은 2블럭 추가하여 진행하고있습니다.
